data/101/749/153/101749153-alt-quattroshapes_pg.geojson

```diff
@@ -0,0 +1,18 @@
+{
+  "id": 101749153,
+  "type": "Feature",
+  "properties": {
+    "src:alt_label":"quattroshapes_pg",
+    "src:geom":"quattroshapes_pg",
+    "wof:geomhash":"30db6336fa8a313c319f58e3f0d17c9e",
+    "wof:id":101749153,
+    "wof:repo":"whosonfirst-data-admin-dn"
```

Comment:
looks like I need to update wof:repo values, too

Reply:
fixed in https://github.com/whosonfirst-data/whosonfirst-data-admin-dk/pull/11/commits/c0e858a1a054baf722c1259564546539830cdfab